homeassistant/components/sensor/buienradar.py

```diff
@@ -185,10 +186,30 @@ def __init__(self, sensor_type, client_name):
         self._attribution = None
         self._measured = None
         self._stationname = None
+        self._unique_id = self.uuid(coordinates)
+
+        # All continuous sensors should be forced to be updated
+        self._force_update = self.type != SYMBOL and \
```

Comment:
What is the update interval of buienradar ? Is it calculated on each call ?

Reply:
The update interval is 10 minutes. And I also added a check whether the measurement is new, so it only gets updated if there really is a new measurement. The `minutes` attribute of the `self._measured` `datetime` object is always a multiple of 10.

Reply:
> I also added a check whether the measurement is new, so it only gets updated if there really is a new measurement

And this means that you cannot use force_update. It means that Home Assistant will keep writing things to the state machine as new measurements while they are not new measurements.

Reply:
Buienradar only releases a new measurement each 10 minutes, so the minimum time between measurements is 10 minutes. By checking whether the measurement is new, I mean checking the measurement time. That means that HA does not keep writing things to the state machine, but only once each 10 minutes.

Reply:
Oh I see now that Buienradar manages its own update cycle. Okay that can work.